libraries/lib-time-and-pitch/StaffPad/TimeAndPitch.h

```diff
@@ -3,17 +3,26 @@
 //
 #pragma once
 
+#include <complex>
+#include <functional>
 #include <memory>
 #include <vector>
+#include <random>
```

Comment:
Why this include is in the header?

Reply:
That's a remnant of me using something more sophisticated. Can be removed I think.